docs/updates/July_2024.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 8
+---
+
+# July 2024
+
+## tl;dr
+
+- Announcing our last raise in the second week of September
+- Continuing to progress on development, will release updated version to testnet
+  next Month
+- Last minute decided to attend Token2049 due to a [PG](https://x.com/PGgroup_)
+  partnership and exclusive event invite
+- Joining two spaces next month, one with Monad, dates and details below
+
+## Development
+
+Interest rate work is close to wrapping up and we are aiming to deploy a new
+version on testnet this month. We have a first draft of the liquidation
+protection that we are working through. The draft is still very early and it is
+going to take me some time to finalize. These are two last massive rocks that we
+have been working on for a while. Getting them done means that we are just
+fine-tuning things and crossing off smaller items. I don’t want to over promise
```

Comment:
"Getting them done means that we are just
fine-tuning things and crossing off smaller items." - this sentence is a bit unclear, getting the big rocks done means we will only have small things left to do, is what you mean -> "Getting them done means that we are left with just
fine-tuning things and crossing off smaller items."